docs/en/api/FS-API.md

```diff
@@ -78,6 +78,36 @@ CreateFilePOptions options = CreateFilePOptions.newBuilder().setBlockSizeBytes(6
 FileOutStream out = fs.createFile(path, options);
 ```
 
+#### Programmatically Modifying Configuration
+
+Alluxio configuration can be set through alluxio-site.properties, but these properties apply to all
+instances of Alluxio that read from the file. If fine-grained configuration management is required,
+pass in a customized configuration object when creating the `FileSystem` object.
+The generated `FileSystem` object will have modified configuration properties, independent of any
+other `FileSystem` clients.
+
+```java
+FileSystem normalFs = FileSystem.Factory.get();
+AlluxioURI normalPath = new AlluxioURI("/normalFile");
+// Create a file with default properties
+FileOutStream normalOut = normalFs.createFile(normalPath);
+...
+out.close();
+
+// Create a file system with custom configuration
+InstancedConfiguration conf = new InstancedConfiguration(ConfigurationUtils.defaults());
+conf.set(PropertyKey.USER_BLOCK_SIZE_BYTES_DEFAULT, "256MB");
```

Comment:
This might not be the best property to use because it can be set on a per-RPC basis anyways. Maybe setting a security parameter to authenticate as a different user instead? Or maybe a write/read block policy?

Reply:
Updated, thanks.